resources/js/pages/Dashboard/Admin/RandomGenerator/Display.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+  <div>
+    <div
+      class="h-screen bg-[url('/images/random-generator/background/comic-yellow.jpg')] bg-cover"
+    >
+      <div v-if="generatorState.state === 'setup'" class="flex h-screen">
+        <div
+          class="m-auto rounded-full bg-gradient-to-r from-pink-600 to-purple-600 bg-clip-text font-['EIGHTY_MILES'] text-[15rem] text-transparent"
+        >
+          ZuFHallsgenerator
+        </div>
+      </div>
+      <div
+        v-if="generatorState.state === 'idle'"
+        class="flex h-screen w-screen flex-col"
+      ></div>
+      <Transition>
+        <div
+          v-if="generatorState.state === 'running'"
+          class="flex h-screen w-screen flex-col overflow-hidden"
+        >
+          <div class="h-screen flex-1 overflow-hidden">
+            <div
+              class="grid h-fit w-screen animate-fly justify-items-center space-y-32 overflow-hidden pt-20"
+            >
+              <Avatar
+                class="animate-wiggle"
+                v-for="user in users"
+                :src="user.avatarUrl"
+                :firstname="user.firstname"
+                :lastname="user.lastname"
+              />
+            </div>
+            <audio autoplay>
+              <source
+                src="/sounds/random-generator/running.mp3"
+                type="audio/mpeg"
+              />
+            </audio>
+            <img
+              class="absolute left-[10%] top-1/2 h-1/3 -translate-y-1/2 transform"
+              src="/images/random-generator/gifs/cat.gif"
+            />
+            <img
+              class="absolute right-[10%] top-1/2 h-1/3 -translate-y-1/2 scale-x-[-1] transform"
+              src="/images/random-generator/gifs/cat.gif"
+            />
+          </div>
+        </div>
+      </Transition>
+
+      <Transition name="winner">
+        <div
+          v-if="generatorState.state === 'stopped'"
+          class="flex h-screen items-center justify-center"
+        >
+          <Avatar
+            class="scale-[130%]"
+            :src="generatorState.user?.avatarUrl"
+            :firstname="generatorState.user?.firstname"
+            :lastname="generatorState.user?.lastname"
+          />
+          <audio autoplay>
+            <source
+              src="/sounds/random-generator/airhorn.mp3"
+              type="audio/mpeg"
+            />
+          </audio>
+          <img
+            class="absolute left-0 top-1/2 h-2/3 -translate-y-1/2 transform"
+            src="/images/random-generator/gifs/trumpet.gif"
+          />
+          <img
+            class="absolute right-0 top-1/2 h-2/3 -translate-y-1/2 scale-x-[-1] transform"
+            src="/images/random-generator/gifs/trumpet.gif"
+          />
+        </div>
+      </Transition>
+    </div>
+  </div>
+</template>
+
+<style>
```

Comment:
The single-file component top-level element order is either script > template > style or, as in our case, template > script > style. See: https://v2.vuejs.org/v2/style-guide/?redirect=true#Single-file-component-top-level-element-order-recommended